File: JS-Front-End/2.JS Functions and Statements Exercise/04. Odd And Even Sum.js
function oddAndEvenSum (num) {
    let oddSum = 0;
    let evenSum = 0;
    let numString = num.toString().split('').map(Number);
    for (const i of numString) {
        if (i % 2 === 0){
            evenSum += i;
        } else {
            oddSum += i;
        }
    }
    return `Odd sum = ${oddSum}, Even sum = ${evenSum}`
}

File: Python - Fundamentals/04. Functions - Exercise/04. Odd and Even Sum.py
# def odd_digits(number: int):
#     pass
#
#
# def even_digits(number: int):
#     pass


num = (input())
new = []
even = []
odd = []
x = 0
l = 0
for i in num:
    if i.isdigit():
        new.append(i)
for j in new:
    if int(j) % 2 == 0:
        even.append(j)
    else:
        odd.append(j)
for k in even:
    x += int(k)
for y in odd:
    l += int(y)
print(f'Odd sum = {l}, Even sum = {x}')


